packages/cli/CHANGELOG.md

```diff
@@ -18,6 +18,16 @@ Changelog entries are classified using the following labels:
 
 - `Removed`: for deprecated features removed in this release
 
+## [1.0.0-rc.11]
+
+### Bumped
+- `epubjs-cli` to version 0.1.6
+- `pagedjs-cli` to version 0.4.3
+
+### Fixed
+- EPUB ...
+- PDF ...
+
```

Comment:
Anders, if we can not find the linked issues that are fixed by the dependency updates we should remove the section from this changelog entry.
```suggestion
```